src/frontend/komponenter/Journalforing/EndreDokumentTittel.tsx

```diff
@@ -0,0 +1,86 @@
+import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import CreatableSelect from 'react-select/creatable';
+import { dokumentTitler } from './konstanter/dokumenttitler';
+
+const StyledKnapper = styled.div`
+    display: flex;
+    justify-content: flex-end;
+    margin-top: 0.5rem;
+`;
+
+const StyledHovedKnapp = styled(Hovedknapp)`
+    margin-right: 0.25rem;
+`;
+
+const StyledDiv = styled.div`
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+`;
+
+const StyledCreatableSelect = styled(CreatableSelect)``;
+
+const customStyles = {
+    container: (styles: Record<string, string>, state: Record<string, string>) => ({
+        ...styles,
+        backgroundColor: '#fff',
+        borderRadius: 4,
+        boxShadow: state.isFocused ? '0 0 0 3px #254b6d' : '',
+        border: '1px solid #78706a',
```

Comment:
Fint hvis vi kan ha alle farger på 1 sted -> styles.ts (dette er noe vi skal rydde opp i behandling også)

Reply:
nav-farger burde kanskje ligge i familie-felles-frontend? Kanskje en egen fil i familie-typer som heter farger.ts eller noe?

Reply:
Burde ikke dette komme fra designsystemet, egentlig?
https://nav-it.slack.com/archives/C7NE7A8UF/p1602845017068300

Reply:
WTF!  Dette ser ut til å funke ut av boksen 🤯
```
import navFarger from 'nav-frontend-core';
navFarger.navBla
```